### PR TITLE
Add task3

### PR DESCRIPTION
**Pull request by AndrewKorol14**

### CODE REVIEW

File: TaskDEV3/EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                int tempDecimalNumber = Int32.Parse(args[0]);
+                int tempSystemBase = Int32.Parse(args[1]);
+            }
+            catch(FormatException)
+            {
+                Console.WriteLine("Exception message: string of decimal number or string of base new numeral system contains wrong symbols or null.");
+                return;
+            }
+
+            if (Convert.ToInt16(args[1])<2||Convert.ToInt16(args[1])>20)
```
Comment (AlexanderSoroka) [inline TaskDEV3/EntryPoint.cs]: codestyle violation

File: TaskDEV3/EntryPoint.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    /// Class EntryPoint
+    /// must get two string arguments, else process exceptions
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                int tempDecimalNumber = Int32.Parse(args[0]);
```
Comment (AlexanderSoroka) [inline TaskDEV3/EntryPoint.cs]: wrong exception processing

File: TaskDEV3/EntryPoint.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                int tempSystemBase = Int32.Parse(args[1]);
+            }
+            catch(FormatException)
+            {
+                Console.WriteLine("Exception message: string of decimal number or string of base new numeral system contains wrong symbols or null.");
```
Comment (AlexanderSoroka) [inline TaskDEV3/EntryPoint.cs]: too complicated error message

File: TaskDEV3/NewNumeralSystemInterpreter.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Text;
+
+namespace TaskDEV3
+{
+    /// <summary>
+    /// Class NewNumeralSystemInterpreter
+    /// interprets decimal number in other numeral system
+    /// </summary>
+    public class NewNumeralSystemInterpreter
```
Comment (AlexanderSoroka) [inline TaskDEV3/NewNumeralSystemInterpreter.cs]: Wrong class name

File: TaskDEV3/NewNumeralSystemInterpreter.cs [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    {
+        int numberInOldSystem;
+        int numeralSystemBase;
+
+        public NewNumeralSystemInterpreter(int oldSystemNumberArgument, int systemBaseArgument)
```
Comment (AlexanderSoroka) [inline TaskDEV3/NewNumeralSystemInterpreter.cs]: useless argument word in argument name

File: TaskDEV3/NewNumeralSystemInterpreter.cs [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            StringBuilder newSystemNumberString = new StringBuilder();
+            int tempDivisionRemainder;
+            do
+            {
+                tempDivisionRemainder = numberInOldSystem % numeralSystemBase;
+                numberInOldSystem = numberInOldSystem / numeralSystemBase;
+                if (tempDivisionRemainder < 10)
+                {
+                    char tempCharSymbol = (char)(tempDivisionRemainder + 48); //using ASCII table, for choosing next numeral in the number of new numeral system
```
Comment (AlexanderSoroka) [inline TaskDEV3/NewNumeralSystemInterpreter.cs]: why +48? useless comments - just rewrite code to make it obvious

File: TaskDEV3/NewNumeralSystemInterpreter.cs [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+        /// <summary>
+        /// Method ReverseNumberInNewSystem
+        /// reverses backward string
+        /// </summary>
+        /// <param name="stringArgument">Backward string</param>
+        /// <returns>Normal string</returns>
+        public string ReverseNumberInNewSystem(string stringArgument)
+        {
+            char[] charArrayForReverse=stringArgument.ToCharArray();
```
Comment (AlexanderSoroka) [inline TaskDEV3/NewNumeralSystemInterpreter.cs]: codestyle violation

Comment (YauheniLoban) [reply TaskDEV3/NewNumeralSystemInterpreter.cs]: You mean that we can implement it more easily ?

File: TaskDEV3/InterpreterInNewSystem.cs [hunk 8]
```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Text;
+
+namespace TaskDEV3
+{
+    /// <summary>
+    /// Interprets decimal number in other numeral system
+    /// </summary>
+    public class InterpreterInNewSystem
```
Comment (AlexanderSoroka) [inline TaskDEV3/InterpreterInNewSystem.cs]: Bad class name

**Pull request merged**